doc/api/errors.md

```diff
@@ -2176,6 +2176,12 @@ size.
 This `Error` is thrown when a read is attempted on a TTY `WriteStream`,
 such as `process.stdout.on('data')`.
 
+<a id="ERR_SYNTHETIC"></a>
```

Comment:
Not sure if this is intended to be placed in the section described as `These errors have never been released, but had been present on master between releases.` Maybe it should be placed in the common section between `ERR_STRING_TOO_LONG` and `ERR_SYSTEM_ERROR` with `###` heading level?